core/src/actions/navigation.ts

```diff
@@ -192,9 +192,13 @@ export class NavigationList extends OptionallyAuthenticatedAction {
       });
     }
 
-    if (navigationMode === "config") {
+    if (navigationMode === "config:unauthenticated") {
+      navigationItems = [{ type: "link", title: "Home", href: "/" }];
```

Comment:
@tealjulia Two questions:

1. Should we add an icon for home, just to be consistent?
2. Do you think we should also have a sign in link?

Reply:
Icon for home for sure.  I'm not sure on the sign in link... going to home just shows them a button for sign in, but maybe?  I'll look at how ui-enterprise does it and mimic that?